docs/Design Document.tex

```diff
@@ -40,8 +40,42 @@ \section{Summary}
 The game is a text-based adventure game (\textit{interactive fiction}), with very simple keyboard-based input. The target play time is \textbf{10---15 minutes}.
 
 \section{Screens}
+
+The opening screen will have an intro title, graphic, and note about the "help" option. The word "help" can be entered at any time to give the player the list of acceptable commands. The player will need to type "Start" when they are ready to begin. 
+
+Each screen following will provide the player with a description of the area, including details about which directions are available to them. Finally, there will be a few areas where the player will "hear something," giving them the option to "listen." These scenes will be important story building points. 
+
 \section{Controls}
+
+Players will be able to enter the following inputs:
+\\
+
+\textbf{Directional}
+\begin{itemize}
+	\item Straight
+	\item Left
+	\item Right
+	\item Turn around \\
+\end{itemize} 
+
+\textbf{Actions}
+\begin{itemize}
+	\item Listen
+	\item Don't Listen
+	\item Pick up \\
+\end{itemize} 
+
+\textbf{Support Menu}
+\begin{itemize}
+	\item Help
+\end{itemize}
+
 \section{Mechanics}
+The game has two basic mechanics: \\
+
+\textbf{Start -> Scene Description -> Direction input -> New Scene Description} \\
```

Comment:
These aren't exactly game mechanics, though they include the mechanic. For this game, the basic mechanic is entering a command or response, pressing enter and the game reacting to that and providing feedback.

What you've written does contain that, though.